### PR TITLE
feat/tool: first implementation

### LINKED ISSUE #2: Implement basic tool (in Rust) to launch 8 vaults to form a single-section local network

### LINKED ISSUE #4: Create a simple User Guide in the README of the repo

### PR DESCRIPTION
**Pull request by bochaco**
Resolves #2 
Resolves #4

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        current_vault_args.push(vault_dir);
+        current_vault_args.push("--hard-coded-contacts");
+        current_vault_args.push(&genesis_contant_info);
+
+        let msg = format!("Launching vault #{}...", i);
+        if args.verbosity > 0 {
+            println!("{}", msg);
+        }
+        debug!("{}", msg);
+        run_vault_cmd(&vault_bin_path, &current_vault_args, args.verbosity)?;
```
Comment (ravinderjangra) [inline src/lib.rs]: This code seems to be repeated between the genesis and other vaults. Maybe we can refactor this and have a single function to do this, something like `launch_vault(is_genesis: bool)`.

Comment (ravinderjangra) [reply src/lib.rs]: I see we have a `run_vault_cmd` function but that doesn't involve parsing the args and generating a final command.

Comment (bochaco) [reply src/lib.rs]: `run_vault_cmd` expects the args to pass to vault. I'll see if I can put the args builder in a helper function as you say

File: README.md [hunk 2]
```diff
@@ -1,6 +1,89 @@
...
+In order to run your own local network you'd need to follow these steps:
+- Download latest release from [safe_vault releases](https://github.com/maidsafe/safe_vault/releases/tag/0.20.1)
+- Untar/unzip the downloaded file into a directory of your choice
+- Execute this tool specifying the path of the `safe_vault` executable
+
+The following is an example of how to perform this on Linux or Mac:
+```shell
+$ mkdir ~/my-local-network
+$ cd ~/my-local-network
+$ wget https://github.com/maidsafe/safe_vault/releases/download/0.21.0/safe_vault-0.21.0-x86_64-unknown-linux-musl.tar.gz
```
Comment (ravinderjangra) [inline README.md]: Should we just use a curl command here? or if somebody tries to use these commands they will have to install the `wget` (at least on mac) before running the command which isn't mentioned here any may confuse them.

**Comment by ravinderjangra** [comment]
@bochaco I think thinking if we should add one more agr here to pass to the vault exec command to specify the vault update or no-update feature?

**Comment by bochaco** [comment]
> @bochaco I was thinking if we should add one more agr here to pass to the vault exec command to specify the vault update or no-update feature?

We can work on such enhancements yes, but as a separate task, also we need to consider that in such case the tool needs to either wait longer for genesis to start or be smarter in the way it checks the log to realise the genesis contact info.

File: README.md (hunk 2 above)
Comment (bochaco) [reply README.md]: ok, I'll change that to curl

**Comment by ravinderjangra** [comment]
> > @bochaco I was thinking if we should add one more agr here to pass to the vault exec command to specify the vault update or no-update feature?
> 
> We can work on such enhancements yes, but as a separate task, also we need to consider that in such case the tool needs to either wait longer for genesis to start or be smarter in the way it checks the log to realise the genesis contact info.

That was my main concern that vault may wait for the long time (if tries to update) then we can pipeline this arg to the vault. Once the vault PR is merged maybe this will more clear since we haven't finalized b/w `--update` and `--no-update`.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,89 @@
 # safe-network-launch-tool
 A cross platform tool to easily launch a SAFE Network test section from home
 
+## Installing the SAFE Vault
+
+A local SAFE network is bootstrapped by running several [SAFE vaults](https://github.com/maidsafe/safe_vault) which automatically interconnect forming a network.
+
+In order to run your own local network you'd need to follow these steps:
+- Download latest release from [safe_vault releases](https://github.com/maidsafe/safe_vault/releases/tag/0.20.1)
```
Comment (S-Coyle) [inline README.md]: Probably best that the link here is changed to `https://github.com/maidsafe/safe_vault/releases/latest/`

File: README.md [hunk 4]
```diff
@@ -1,6 +1,89 @@
...
+Launching vault #3...
+Launching vault #4...
+Launching vault #5...
+Launching vault #6...
+Launching vault #7...
+Launching vault #8...
+Done!
+```
+
+Once the local network is running, the connection configuration file will be already in the correct place for your applications to connect to this network, so you can simply any application from this moment on to connect to your local network. Note that depending on the application, you may need to restart it so it uses the new connection information for your local network.
```
Comment (S-Coyle) [inline README.md]: > so you can simply any application
Looks like the word `run` was missed in the middle of this

File: README.md [hunk 5]
```diff
@@ -1,6 +1,89 @@
...
+
+In order to shutdown a running local network, all processes instances of safe_vault must be killed, e.f. on Linux or Mac you can use the `killall` command:
```
Comment (S-Coyle) [inline README.md]: Should be `e.g.` instead of `e.f.`

File: README.md (hunk 3 above)
Comment (bochaco) [reply README.md]: yes, true, I'll add that change

File: README.md (hunk 4 above)
Comment (bochaco) [reply README.md]: yes, thanks I'll correct that

File: README.md [hunk 6]
```diff
@@ -1,6 +1,89 @@
...
+
+USAGE:
+    safe-nlt [FLAGS] [OPTIONS]
+
+FLAGS:
+    -h, --help                
+            Prints help information
+
+    -y, --vaults-verbosity    
+            Vebosity level for vaults logs (default: INFO)
```
Comment (S-Coyle) [inline README.md]: Missed the `r` in Verbosity

File: README.md [hunk 7]
```diff
@@ -1,6 +1,89 @@
...
+
+    -V, --version             
+            Prints version information
+
+    -v, --verbosity           
+            Vebosity level for this tool
```
Comment (S-Coyle) [inline README.md]: Missed the `r` in Verbosity

**Pull request merged**